src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx

```diff
@@ -170,7 +170,7 @@ These queries apply for some older New Relic organizations that have only two us
 
     ### Projected monthly full platform user count
 
-    This query shows a projected count of monthly full platform users. This query would not be good for using in a dashboard; it requires values based on a) the days remaining in the month, b) the start of the month. Here's an example querying the projected end-of-month count with 10 days left in that month:
+    This query shows a projected count of monthly full platform users. This query would not be good for using in a dashboard; it requires values based on the days remaining in the month, and the start of the month. Here's an example querying the projected end-of-month count with 10 days left in that month:
```

Comment:
```suggestion
    Here's an example of querying the projected end-of-month count of monthly full platform users with 10 days left in that month. Note that this query would not be good for using in a dashboard because it requires values based on the days remaining in the month and the start of the month. 
```

Reply:
@brnhensley I rejiggered this in a way that made more sense to me. Is this revision accurate?

Reply:
I believe so